### ISSUE TITLE
Unable to build

### ISSUE THREAD

**Issue by Oren-Schmerling** [reportportal/plugin-slack#7]
**tldr:** I want to customize the slack plugin, but I can't build it locally. I've tried playing around with the commit versions of commons-dao and plugin-api dependencies with no success. 

I've cloned the repo, ran `npm i` within `/plugin-slack/ui`, and then building with `./gradlew build` gives the following error: 
```
> Task :plugin-slack:compileJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':plugin-slack:compileJava'.
> Could not resolve all files for configuration ':plugin-slack:compileClasspath'.
   > Could not resolve com.github.reportportal:plugin-api:188792e.
     Required by:
         project :plugin-slack
      > Could not resolve com.github.reportportal:plugin-api:188792e.
         > Could not get resource 'https://nodejs.org/dist/v188792e/plugin-api.jar'.
            > Could not HEAD 'https://nodejs.org/dist/v188792e/plugin-api.jar'. Received status code 403 from server: Forbidden
```

After seeing this, I thought that it might be trying to build using an outdated release, so I changed the commits to point to the most recent release version that existed on both commons-dao and plugin-api (5.12.0). This resulted in a similar error:

```
* What went wrong:
Execution failed for task ':plugin-slack:compileJava'.
> Could not resolve all files for configuration ':plugin-slack:annotationProcessor'.
   > Could not find com.github.reportportal:commons:9620556.
     Required by:
         project :plugin-slack > com.github.reportportal:plugin-api:b5ddb40
   > Could not find com.github.reportportal:commons:9620556.
     Required by:
         project :plugin-slack > com.github.reportportal:plugin-api:b5ddb40 > com.github.reportportal:commons-dao:823f284
```

This seems strange since that commit points to a mergerd [PR](https://github.com/reportportal/commons/pull/27) that's quite a few commits behind the release of 5.12.0 in the commons repo.

I also tried simply replacing the comit IDs with the name of the desired version, but that yielded the same result: 
```
    } else {
     -  implementation 'com.github.reportportal:commons-dao:acf1ec7'
     -  implementation 'com.github.reportportal:plugin-api:188792e'
     -  annotationProcessor 'com.github.reportportal:plugin-api:188792e'
     + implementation 'com.github.reportportal:commons-dao:5.12.0'
     + implementation 'com.github.reportportal:plugin-api:5.12.0'
     + annotationProcessor 'com.github.reportportal:plugin-api:5.12.0'
    }
```
running gradlew build then gave:
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':plugin-slack:compileJava'.
> Could not resolve all files for configuration ':plugin-slack:compileClasspath'.
   > Could not find com.github.reportportal:commons-dao:5.12.0.
     Required by:
         project :plugin-slack
   > Could not find com.github.reportportal:commons-dao:5.12.0.
     Required by:
         project :plugin-slack > com.github.reportportal:plugin-api:5.12.0
```
I also tried using the newest versions of commons-dao (5.12.1) with the newest version of plugin-api (5.12.0), but that also didn't work:
```
* What went wrong:
Execution failed for task ':plugin-slack:compileJava'.
> Could not resolve all files for configuration ':plugin-slack:annotationProcessor'.
   > Could not find com.github.reportportal:commons:9620556.
     Required by:
         project :plugin-slack > com.github.reportportal:plugin-api:b5ddb40
   > Could not find com.github.reportportal:commons:9620556.
     Required by:
         project :plugin-slack > com.github.reportportal:plugin-api:b5ddb40 > com.github.reportportal:commons-dao:823f284
```
I tried using `releaseMode = true` hoping that would correctly resolve the dependencies, but that resulted in the exact same error message as above.
